### ISSUE TITLE
fstdeterminizestar nnet3 model(not chain)

### ISSUE THREAD

**Issue by YangangCao** [alphacep/vosk-api#1509]
Hi, dear author

when I ran ./local/lookahead/run_lookahead_no_ivector.sh in a nnet3 model to get vosk-like model, it cost all memory(more than 450GB) in server, and killed, it stuck in fstdeterminizestar.
And I change nnet3 model to chain model, it become normal. Any idea to fstdeterminizestar nnet3 model or make nnet3 model lookahead?

**Comment by nshmyrev** [comment]
Yeah, overall determinization is like magic. It can fail sometimes. Some FST experts might help, but unlikely I can suggest you something quick right now, sorry.
